contrib/marketplace/README_zh_CN.md

```diff
@@ -0,0 +1,58 @@
+# Marketplace Plugin
```

Comment:
could you do not add English "zh_CN.md" files until Chinese version is available?

Reply:
It's using a same rule to copy files. I will try to remove them, if they are out of docs folder. For the files in docs folder, they will be kept to keep links works cross file.